### PR TITLE
Admin tool, the life candle!

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
add: Added an admin only tool, the life candle. Touch the candle, and
when you die, you'll respawn shortly afterwards. Touch it again to stop.
Used for testing, thunderdome brawls and good old fashioned memery.
/:cl:

Requested.

### CODE REVIEW

**Comment by coiax** [comment]
@Aloraydrel

**Comment by Durkel** [comment]
make it a bonfire for memes

**Comment by Aloraydrel** [comment]
Finally I can make ss13 pudge wars

**Comment by coiax** [comment]
@Aloraydrel if you want to suggest some presets, I can make some convenience subtypes.

**Comment by FantasticFwoosh** [comment]
Something fixed to the floor that can't be destroyed or picked back up again. (to stop people running away from your checkpoint or otherwise destroying it) without admin privileges.

Pulling the sword from the stone so to speak.

**Comment by coiax** [comment]
@FantasticFwoosh it just looks like a candle, it's an anchored structure.

File: code/modules/admin/life_candle.dm [hunk 1]
```diff
@@ -0,0 +1,90 @@
+/obj/structure/life_candle
```
Comment (Cyberboss) [inline code/modules/admin/life_candle.dm]: Wrong location. Put it in code/game/objects/structures

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by optimumtact** [comment]
What could possibly go wrong

**Comment by coiax** [comment]
If anyone goes wrong, it's the admins fault

**Pull request merged**